pulsar-io/flume/src/test/java/org/apache/pulsar/io/flume/node/TestEnvVarResolverProperties.java

```diff
@@ -18,13 +18,14 @@
  */
 package org.apache.pulsar.io.flume.node;
 
-import java.io.File;
```

Comment:
Please check that you have proper code styling in place. https://github.com/apache/pulsar/#configure-code-style . The Pulsar code style doesn't group the imports separately.

Reply:
Ok, i fomatted the code style.

Reply:
ping @lhotari, PTAL again, thanks.